addons/nuxeo-platform-document-routing/nuxeo-routing-api/src/main/java/org/nuxeo/ecm/platform/routing/api/DocumentRoutingService.java

```diff
@@ -586,10 +586,15 @@ void grantPermissionToTaskDelegatedActors(CoreSession session, String permission
     /**
      * @param userId
      * @param workflowInstanceId
+     * @param workflowInstanceName
```

Comment:
It's not needed, javadoc `@param` makes sense when there's documentation. All blank lines should be cleaned from our code or at least not introduced.

Here, as the method is deprecated it's not a big deal.